### ISSUE TITLE
[Feature Request] Comment thread text search

### ISSUE THREAD

**Issue by Jaegrqualm** [michael-lazar/rtv#335]
I often want to search a comment thread to see if someone mentioned a topic. As is, I don't see any way to do that.

**Comment by michael-lazar** [comment]
Good idea. I think we could either try to do this natively or pipe everything to ``less`` which has a built-in search function. One thing that might be a problem is that usually only the first 200ish comments are loaded and there would be no way to search *all* of the comments.

**Comment by Jaegrqualm** [comment]
There's an account setting for the number of comments, which has a limit
increase with reddit gold. I don't know if rtv obeys that right now, but
that could end up being a separate feature request.

**Comment by michael-lazar** [comment]
Yea I believe that we don't specify the number of comments to fetch, which means that it falls back to the user's default setting.

https://github.com/praw-dev/praw/blob/praw3/praw/objects.py#L1131

**Comment by indeedwatson** [comment]
Chiming in to say I really want this too, and I think it would be ideal if the same bind that's used to search across subreddits was used to search for words when you're inside a submission.
